README.md

```diff
@@ -95,4 +100,4 @@ This client is tested with Minitest.
 Please run all tests before submitting a Pull Request, and add new tests for new functionality.
 
 Running tests: <br>
-`rake test`
\ No newline at end of file
+`rake test`
```

Comment:
what's different in the last line (103) - why does it show up as a diff? I am looking at it and it looks the same to me :)